source/UF/Pushouts.lagda

```diff
@@ -0,0 +1,418 @@
+Ian Ray, 15th January 2025
+
+Edited by Ian Ray on 16th March 2025.
+
+Pushouts are defined as a higher inductive type (in the form of a record type). We
+postulate point and path constructors and the (dependent) universal property.
+We will derive other important results like induction and recursion principles along
+and the corresponding propositional computation rules. Of course, due to Kristina
+Sojakova's dissertation (and the following paper on the same topic doi:
+https://doi.org/10.1145/2775051.2676983) it is well known that for higher inductive
+types with propositional computation rules the following are equivalent:
+
+1) dependent homotopy initiality
+2) induction principle with propositional computation rules
+3) recursion principle with propositional computation rules and uniqueness
+   principle
+4) non-dependent homotopy initiality
+
+Sojakova uses the term 'homotopy initiality' for the universality of 'algebra
+morphisms' (see the paper above). Here we choose a weaker phrasing in terms of the
```

Comment:
I think you can leave out "(see paper above)" as this is clearly implied by your discussion.